Dockerfile.dapper

```diff
@@ -19,6 +19,7 @@ RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core w
 RUN zypper install -y -f docker && rpm -e --nodeps --noscripts containerd
 
 RUN curl -sLf https://github.com/rancher/machine/releases/download/${CATTLE_MACHINE_VERSION}/rancher-machine-${ARCH}.tar.gz | tar xvzf - -C /usr/bin
+RUN curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
```

Comment:
This will work but can lead to unexpected breaking if a new version releases which has issues or is not backwards compatible.

Reply:
Are you suggesting, downloading the specific version instead?